src/components/domain/ReservationItem/ReservationItemBottom/index.tsx

```diff
@@ -45,21 +51,22 @@ const ReservationItemBottom = ({
       </Container>
       {visible && (
         <ParticipantList>
-          {participants.length !== 0 ? (
-            participants.map(
-              ({ userId, nickname, profileImage, isFollowed }: any) => (
-                <FollowListItem
-                  key={userId}
-                  src={profileImage}
-                  isFollowed={isFollowed}
-                  userId={userId}
-                >
-                  {nickname}
-                </FollowListItem>
-              )
+          {userId && profileImageUrl && (
+            <FollowListItem userId={userId} src={profileImageUrl}>
+              {nickname}
+            </FollowListItem>
+          )}
```

Comment:
이렇게 하면 어떨까요? profileImageUrl은 로그인된 유저의 프로필이 null일 수도 있어요
그래서 null인 프로필이미지를 가지고 있는 유저는 여기에 FollowListItem이 보이지 않을 거에요
그럼 잘못 동작되는 거겠죠?

```suggestion
          {userId && (
            <FollowListItem userId={userId} src={profileImageUrl || "/assets/default_profile.svg"}>
              {nickname}
            </FollowListItem>
          )}
```

Reply:
FollowListItem 의 src가 string만 받을 수 있게 되어 있네요~
profileImageUrl || 'defaultUrl을' 일단 넣는 것으로 해결합시다
이슈에 Profile은 domain Component으로 분리하는 작업 추가해놓겠습니다

Reply:
제안 받아주세요 approve할게요